dockstore-integration-testing/src/test/java/io/dockstore/webservice/UserResourceIT.java

```diff
@@ -730,4 +729,37 @@ public void testUpdateUserMetadataWithoutTokens() {
         assertNull(userProfile.getAvatarURL());
         assertNull(userProfile.getLocation());
     }
+
+    @Test
+    public void testGetStarredWorkflowsAndServices() throws Exception {
+        CommonTestUtilities.cleanStatePrivate2(SUPPORT, false);
+        final io.dockstore.openapi.client.ApiClient webClient = getOpenAPIWebClient(USER_2_USERNAME, testingPostgres);
+        final io.dockstore.openapi.client.api.WorkflowsApi workflowsApi = new io.dockstore.openapi.client.api.WorkflowsApi(webClient);
+        final io.dockstore.openapi.client.api.UsersApi usersApi = new io.dockstore.openapi.client.api.UsersApi(webClient);
+        final long userId = usersApi.getUser().getId();
+
+        // Add service
+        workflowsApi.handleGitHubRelease("refs/tags/1.0", INSTALLATION_ID, SERVICE_REPO, USER_2_USERNAME);
+        assertEquals(1, usersApi.userServices(userId).size());
+        assertEquals(0, usersApi.userWorkflows(userId).size());
+
+        // Star service
+        io.dockstore.openapi.client.model.Workflow service = workflowsApi.getWorkflowByPath("github.com/" + SERVICE_REPO, SERVICE, "");
+        io.dockstore.openapi.client.model.StarRequest starRequest = new io.dockstore.openapi.client.model.StarRequest().star(true);
+        workflowsApi.starEntry1(service.getId(), starRequest);
```

Comment:
why did this and the refresh become numbered?

Reply:
I'm not sure. There are other methods in the openapi workflowsapi that have a number at the end too, like `deleteTestParameterFiles1`

Reply:
Looks like there's a conflict between the openapi `operationId` and swagger `nickname`: https://github.com/dockstore/dockstore/blob/b09d16c7501d40159a030f30bde59a8aaf18c183/dockstore-webservice/src/main/resources/openapi3/openapi.yaml#L6368 
https://github.com/dockstore/dockstore/blob/b09d16c7501d40159a030f30bde59a8aaf18c183/dockstore-webservice/src/main/resources/swagger.yaml#L5944

I tried removing the swagger `nickname`, but it doesn't compile because there are some tests that use the swagger api client https://github.com/dockstore/dockstore/blob/b09d16c7501d40159a030f30bde59a8aaf18c183/dockstore-webservice/src/main/java/io/dockstore/webservice/resources/WorkflowResource.java#L1543